stewart/inc/patterns/sidebar-border.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Sidebar with border block pattern
+ */
+return array(
+	'title'      => __( 'Sidebar with border', 'stewart' ),
+	'categories' => array( 'sidebar' ),
+	'content'    => '<!-- wp:group {"style":{"spacing":{"padding":{"top":"2em","right":"2em","bottom":"2em","left":"2em"}},"border":{"width":"1px"}}} -->
+					<div class="wp-block-group" style="border-width:1px;padding-top:2em;padding-right:2em;padding-bottom:2em;padding-left:2em"><!-- wp:site-logo {"width":90,"align":"center","className":"is-style-rounded"} /-->
+
+					<!-- wp:site-title {"textAlign":"center","style":{"typography":{"lineHeight":"1.2"}},"fontSize":"gigantic"} /-->
+
+					<!-- wp:site-tagline {"textAlign":"center","fontSize":"small"} /-->
+
+					<!-- wp:separator {"className":"is-style-wide"} -->
+					<hr class="wp-block-separator is-style-wide"/>
+					<!-- /wp:separator -->
+
+					<!-- wp:navigation {"ref":99,"overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"center","orientation":"vertical"},"style":{"spacing":{"blockGap":"0px"}}} /-->
```

Comment:
I think this "ref":99 is causing a "menu has been deleted or is unavailable" error

![image](https://user-images.githubusercontent.com/146530/150361775-33950842-ac0d-4853-84f6-929baf4eb9fe.png)

Reply:
Ah, good catch. Should be fixed in 5ac6604.